src/parser/jobs/mnk/modules/DISPLAY_ORDER.ts

```diff
@@ -1,7 +1,8 @@
 export const DISPLAY_ORDER = {
 	RIDDLE_OF_FIRE: 2,
 	PERFECT_BALANCE: 3,
-	DRAGON_KICK: 5,
-	TWIN_SNAKES: 6,
-	DEMOLISH: 7,
+	RIDDLE_OF_WIND: 4,
+	DRAGON_KICK: 6,
+	TWIN_SNAKES: 7,
+	DEMOLISH: 8,
```

Comment:
Not sure why this needs to happen, statistic boxes are fixed locations and without any suggestions there's nothing to display so the display order for RoW isn't used. Generally also prefer to not re-order this file.